apps/frontend/src/components/common/UserManagementTable.tsx

```diff
@@ -54,6 +55,7 @@ const UserManagementTable = ({
   const isLegacyInviteFlow = featuresMap.get(
     FeatureId.EMAIL_INVITE_LEGACY
   )?.isEnabled;
+  const currentUser = useContext(UserContext)?.user;
```

Comment:
question/suggestion: Isn't the useContext(UserContext) always defined? So we could simply have
```suggestion
  const currentUser = useContext(UserContext).user;
````